server/src/main/scala/com/metamx/tranquility/server/config.scala

```diff
@@ -69,10 +70,41 @@ object config
     @Config(Array("tranquility.lingerMillis"))
     def tranquilityLingerMillis: Int = Tranquilizer.DefaultLingerMillis
 
+    @Config(Array("druidBeam.firehoseGracePeriod"))
+    def firehoseGracePeriod: Period = DruidBeamConfig().firehoseGracePeriod
+
+    @Config(Array("druidBeam.firehoseQuietPeriod"))
+    def firehoseQuietPeriod: Period = DruidBeamConfig().firehoseQuietPeriod
+
+    @Config(Array("druidBeam.firehoseRetryPeriod"))
+    def firehoseRetryPeriod: Period = DruidBeamConfig().firehoseRetryPeriod
+
+    @Config(Array("druidBeam.firehoseChunkSize"))
+    def firehoseChunkSize: Int = DruidBeamConfig().firehoseChunkSize
+
+    @Config(Array("druidBeam.randomizeTaskId"))
+    def randomizeTaskId: Boolean = DruidBeamConfig().randomizeTaskId
+
+    @Config(Array("druidBeam.indexRetryPeriod"))
+    def indexRetryPeriod: Period = DruidBeamConfig().indexRetryPeriod
+
+    @Config(Array("druidBeam.firehoseBufferSize"))
+    def firehoseBufferSize: Int = DruidBeamConfig().firehoseBufferSize
+
     @Config(Array("druid.discovery.curator.path"))
     def discoPath: String = "/druid/discovery"
 
     override def discoAnnounce: Option[DiscoAnnounceConfig] = None
+
+    def druidBeamConfig: DruidBeamConfig = DruidBeamConfig.builder()
```

Comment:
Something crazy will probably happen if you use a negative chunk size or buffer size. If we want to validate that kind of stuff then DruidBeamConfig is probably the right place to do it